tests/Fixtures/inc/Engine/Cache/WPCache/maybePreventDeactivation.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+	'vfs_dir' => 'public/',
+
+	'test_data' => [
+		[
+
+		]
+	]
+];
```

Comment:
@Tabrisrp @hellofromtonya Is there a better way to do this? We don't really need any testdata for this one, but we do need to use the filesystem, so we're extending the FilesystemTestCase (which seems to require the fixture to establish the `vfs_dir`.

Reply:
In your test file, you can group up the different testing methods into one, and use the data provided from here to change the way the test behave. For example:

```
testShouldBailOutWhenConfigFileFound => [
    'expected' => [],
],
testShouldBailOutWhenSetCacheConstFilterFalse => [
    'set_constant' => false,
    'expected'       => [],
],
testShouldAddCauseToCausesWhenNotPrevented => [
    'file_exist'   => false,
    'expected'  => [ 'config' ],
],
```

You can check the existence of the array keys to perform different actions in the test code and get the expected result